ur_description/package.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
```

Comment:
```suggestion
<package format="2">
```

I think for now it makes sense to stick with package format 2 until https://github.com/ros-infrastructure/rep/pull/347 is resolved.

Reply:
As far as I remember, the reason why we change the format version was to support multiple licenses

Reply:
That's not supported, yet, hence it is in the commented line. Having multiple license tags is also possible in format 2.

Reply:
As far as I remember then the reason for changing the package format, is to be able to use the following in the license.
```
<license file="meshes/ur20/LICENSE.txt">Universal Robots A/S' Terms and Conditions for Use of Graphical Documentation</license>
```
 The license file is not supported in package format 2 only in 3.

Reply:
I remembered that exactly that was supported by format 2, as well. I checked the specs and it seems I remembered that incorrectly.